### PR TITLE
Update array_helpers.py

### LINKED ISSUE #15793: [Bug]: ivy/ivy_tests/test_ivy/helpers/hypothesis_helpers /array_helpers.py need to be updated
### Bug Explanation

The helper folder contain array helper program in test_ivy folder contains a small error when a array of dimension 3 is used then it is showing an error of "Local Error The variable kernel is referenced before use".

### Steps to Reproduce Bug

When a array of dimension 3 is used and any test case related to array_pooling then the test is failing showing a error of LocalError.

### Environment

When adding a new function to stateful classes called max_pool1d as well as the test_layers for writing a test case for max_pool1d class. Then the error is encountered when test cases are running in which the jax test case failed showing the error.

### Ivy Version

v1.1.9

### Backend

- [ ] NumPy
- [ ] TensorFlow
- [ ] PyTorch
- [X] JAX

### Device

_No response_

### PR DESCRIPTION
**Pull request by mtnvdsk**
closes #15793

### CODE REVIEW

File: ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py [hunk 1]
```diff
@@ -1555,7 +1555,7 @@ def arrays_for_pooling(
         )
     if array_dim == 3:
         kernel = draw(st.tuples(st.integers(1, in_shape[1])))
-    new_kernel = kernel
+        new_kernel = kernel
```
Comment (CatB1t) [inline ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py:1558]: Hello @mtnvdsk , this will actually make it fail if the array dim is not 3, it shouldn't be indented.

**Review by CatB1t — CHANGES_REQUESTED** [review]

**Comment by mtnvdsk** [comment]
@CatB1t sir, Updated the code and all the test cases are passed.

**Comment by mtnvdsk** [comment]
@CatB1t sir, can you please review any further changes in the code because I need to do another PR in the ivy to add a max_pool1d code which is failing due to this dimension 3 in the array helpers code.

**Comment by mtnvdsk** [comment]
@Daniel4078 sir,can I know why this pull request is closed?

**Comment by Daniel4078** [comment]
> @Daniel4078 sir,can I know why this pull request is closed?

I was thinking that it maybe better to try solve this problem in the pull request of yours that is affect this potential bug. As this pull request is not properly linked to anything with no tags or discription of the problem.

**Comment by mtnvdsk** [comment]
But @CatB1t sir asked me to do a separate PR for this issue so please reopen this PR @Daniel4078

**Review by CatB1t — CHANGES_REQUESTED** [review]
Hello @mtnvdsk , can you please explain the rationale behind those changes? what is the bug/issue you're trying to fix?

**Comment by mtnvdsk** [comment]
@CatB1t sir, I thought the dimension parameter is hard-coded for some sizes so to reduce it as well as the size 3 dimension error causing.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.
